CHANGES

```diff
@@ -9,6 +9,8 @@
 
  Changes between 1.1.0h and 1.1.1 [xx XXX xxxx]
 
+  *) SSL_MODE_AUTO_RETRY is enabled by default.
```

Comment:
Normally a CHANGES entry would have your name underneath it. Also, I think this deserves a bit more commentary than this. Probably some justification for the change. Perhaps a pointer to the documentation. Perhaps a hint on how to turn this off.